setup.py

```diff
@@ -60,7 +60,7 @@ def build_extensions(self):
         dimod.get_include(),
         ],
     install_requires=[
-        'numpy>=1.17.3,<2.0.0',  # keep synced with circle-ci, pyproject.toml
-        'dimod>=0.10.0,<0.11.0'
+        'numpy>=1.19.1,<2.0.0',  # keep synced with circle-ci, pyproject.toml
+        'dimod>=0.11.0rc0,<0.12.0'
```

Comment:
IIRC, `>=...rc` will trigger pip's `--pre` (for other packages as well).

Reply:
Hmm, not sure there's much to be done about that. I think it's relatively safe in this case since it's just NumPy?

Reply:
Maybe all other packages if installed as part of Ocean? Not sure. But we can test and update if needed.

Reply:
One obvious thing is just release this package as an `.rc` and and then when we release a non-`.rc` version of dimod we also update this one.

Reply:
And only the non-`.rc` will make it into a proper sdk release.